### PR TITLE
Docs/integrations

### PR DESCRIPTION
**Pull request by Mcastres**
#### Description of what you did:

Added integrations for:
  - Gridsome
  - Jekyll
  - Hugo
  - 11ty
  - Svelte
  - Sapper
  - Ruby
  - Python
  - Angular

### CODE REVIEW

**Comment by derrickmehaffy** [comment]
@Mcastres first off, **holy cow**, second nice job.

Give me a few days to review ;)

**Review by derrickmehaffy — APPROVED** [review]
LGTM, one suggestion I might make, it may be easier to just copy the SVG binaries into the docs folder and link them instead of dumping their entire contents into the theme component.

File: docs/v3.x/getting-started/ruby.md
Comment (soupette) [inline docs/v3.x/getting-started/ruby.md:12]:
```bash
mkdir ruby-app && cd ruby-app
touch script.rb
```

File: docs/v3.x/getting-started/ruby.md
Comment (soupette) [inline docs/v3.x/getting-started/ruby.md:76]: Can you describe the file structure of this app?

I think we need a Gemfile as well to install `gems` so they are not installed globally.

Comment (soupette) [reply docs/v3.x/getting-started/ruby.md:76]: Can you make sure this app follows the same views as the others so all exemples are consistent?

File: docs/v3.x/getting-started/svelte.md
Comment (soupette) [inline docs/v3.x/getting-started/svelte.md:304]: Same here about the example. Why don't we have it for the other apps?

**Review by soupette — CHANGES_REQUESTED** [review]
It would be great if all the apps had the same features. Can you make sure they all follow the same logic?

File: docs/v3.x/getting-started/angular.md
Comment (Mcastres) [reply docs/v3.x/getting-started/angular.md:138]: We'll leave it this way, it's a example, don't want to confuse them with env vars

File: docs/v3.x/getting-started/hugo.md
Comment (Mcastres) [reply docs/v3.x/getting-started/hugo.md:68]: The documentation is very poor in my opinion and I'm not confortable with it. not sure to be able to reproduce correctly using the best practice the content of the other pages

File: docs/v3.x/getting-started/jekyll.md
Comment (Mcastres) [reply docs/v3.x/getting-started/jekyll.md:62]: Don't understand this one :/

File: docs/v3.x/getting-started/ruby.md
Comment (Mcastres) [reply docs/v3.x/getting-started/ruby.md:76]: I think the less configuration file there are the easier it will be for the user so just installing the gem globally is a good thing instead of having a Gemfile installed etc...

File: docs/v3.x/getting-started/svelte.md
Comment (Mcastres) [reply docs/v3.x/getting-started/svelte.md:304]: Difficult to reproduce on specific tech I'm not comfortable with

File: docs/v3.x/getting-started/jekyll.md
Comment (soupette) [reply docs/v3.x/getting-started/jekyll.md:62]: Updated! sorry about the wrong copy/paste...

File: docs/v3.x/getting-started/ruby.md
Comment (soupette) [reply docs/v3.x/getting-started/ruby.md:76]: Actually as developer point of view I really don't like to install gems or modules globally.

**Comment by Mcastres** [comment]
Can you please review this when you have time @lauriejim @alexandrebodin

File: docs/v3.x/getting-started/svelte.md
Comment (derrickmehaffy) [reply docs/v3.x/getting-started/svelte.md:304]: Would love to help here also but I'm not familiar with svelte either

File: docs/v3.x/getting-started/python.md
Comment (derrickmehaffy) [inline docs/v3.x/getting-started/python.md:105]: wrong language highlighted?

**Review by derrickmehaffy — CHANGES_REQUESTED** [review]

File: docs/v3.x/getting-started/python.md
Comment (Mcastres) [reply docs/v3.x/getting-started/python.md:105]: You're right

**Comment by Mcastres** [comment]
Hey everyone! It would be great to merge this if there is no more change requests here @lauriejim @derrickmehaffy

**Comment by derrickmehaffy** [comment]
I'll take another look today or tmrw

**Review by derrickmehaffy — APPROVED** [review]
LGTM

**Review by lauriejim — APPROVED** [review]
LGTM

**Comment by lauriejim** [comment]
@Mcastres Need approval from one of the tech team to be sure it has been reviewed and approved on the tech side.
Then, ready to merge and deploy.

**Comment by Mcastres** [comment]
Is it good for you? @soupette

**Comment by soupette** [comment]
> Is it good for you? @soupette

No it is not I have requested changes that have been put in resolved but I strongly believe that they should be added, especially the ones where the doc states to install dependencies globally.

**Comment by Mcastres** [comment]
I resolved it because I made the modifications. You can check it out in the Ruby file

**Review by alexandrebodin — CHANGES_REQUESTED** [review]
Generaly speaking it is ok. plz apply the comments in the rest of the PR I just didn't copy/paste everywhere.

The fact that multiple examples are not consistent in terms of features is worrying me as you are going to get a lot more questions @derrickmehaffy @lauriejim. I think we should release a bunch of the most known ones that are really consistent and pass specific PRs for the rest of them to get help by community contributors that will be able to complete Maxime's work.

**Comment by Mcastres** [comment]
Thanks @alexandrebodin for taking the time to review this PR!

File: docs/v3.x/getting-started/angular.md
Comment (alexandrebodin) [inline docs/v3.x/getting-started/angular.md:71]: You went a bit too far :) console.error is only in  necessary in the catch ;)

**Review by alexandrebodin — CHANGES_REQUESTED** [review]
a small precision ;)

File: docs/v3.x/getting-started/angular.md
Comment (Mcastres) [reply docs/v3.x/getting-started/angular.md:71]: My editor changed all of them...
I'll fix it

Comment (Mcastres) [reply docs/v3.x/getting-started/angular.md]: Done!

**Review by alexandrebodin — APPROVED** [review]
Good step for a first version :)

**Pull request merged**
